packages/tools/accessibility/README.md

```diff
@@ -0,0 +1,88 @@
+
+### RenderAccessibilityTree()
+
+The accessibility package basically generates HTML elements for your scene, based on the metadata (IAccessibilityTag) you added to your scene content. The screen reader then can read the HTML elements. To generate it for your scene:
+
+```javascript
+ACCESSIBILITY.AccessibilityRenderer.RenderAccessibilityTree(scene);
+```
+
+This will generate a `<div id="accessibility-tree">` HTML element right after the babylonJS scene's canvas element. Inside this div element, the renderer generate an HTML twin for each of your accessible contents in the scene. These HTML twin elements are internally connected with BabylonJS objects, so the screen reader user can also interact with the BabylonJS objects through their HTML twin.
+
+### Interaction
+
+Some of your contents in the scene might be interactable (e.g. clickable). For Node type objects, if you use BabylonJS's [ActionManager](https://doc.babylonjs.com/divingDeeper/events/actions) to define the interaction, the Accessibility package can automatically detects it and apply on the generated HTML twin elements, so that the user can use keyboard to trigger events like click or right click on the HTML twin elements, thus trigger the correspond action on the BabylonJS objects. Only ACTION_OnPickTrigger, ACTION_OnLeftPickTrigger, ACTION_OnRightPickTrigger are supported. For Control type objects, if you defined observer for onPointerClickObservable, the Accessibility package can also automatically detects it and apply on the generated HTML twin element.
+
+If you want to customize the interaction, use the eventHandler field of IAccessibilityTag:
+
+```javascript
+let egg = BABYLON.MeshBuilder.CreateSphere("Egg", {diameterX: 0.62, diameterY: 0.8, diameterZ: 0.6}, scene);
+egg.accessibilityTag = {
+    description: "An easter egg",
+    eventHandler: {
+      "onclick": yourFunction
+    }
+}
+```
+Example:
+https://playground.babylonjs.com/?snapshot=refs/pull/12074/merge#C9GZTF#6
+
+### When is AccessibilityTree Updated
+Your scene might be not static, and you may want to update the accessibility tree when your scene is changed. The accessibility tree will automatically update when:
+
+- A Node (Mesh or TransformNode) is added/removed in a scene;
+- A Node (Mesh or TransformNode)'s enabled status is changed;
+- A Control is added/removed from a Container;
+- A Control's isVisible status is changed;
+- A Node or Control's IAccessibilityTag is assigned or re-assigned;
+
+Example:
+https://playground.babylonjs.com/?snapshot=refs/pull/12074/merge#C9GZTF#5

```

Comment:
this will need to be converted to a real playground link once the PG is updated the accessibility is live.